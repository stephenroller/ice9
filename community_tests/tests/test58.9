# break w/in fa in proc

proc foo()
	fa i := 1 to 10 ->
		if i = 7 ->
			break;
		fi
		
		write i;
	af
end

foo();
write 11;