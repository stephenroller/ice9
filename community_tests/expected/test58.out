Loading file: generated/test58.tm
1 
2 
3 
4 
5 
6 
11 
Number of instructions executed = 266
